www/src/app/chat/layout.tsx

```diff
@@ -1,3 +1,6 @@
+"use client";
```

Comment:
what changed for us to need use client here? 🤔

Reply:
We added the ChatSidebar inside this file, then it's necessary to have the "use client"